modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
+You can add different extensions to your plug-in that are loaded at runtime.
+
+[id="adding-tab-to-pods-page"]
+== Adding a tab to the pods page
+You can add a tab to the *Pod Details* page.
+
+.Procedure
+
+. Edit the `console-extensions.json` file.
++
+[source,json]
+
+----
+}
```

Comment:
```suggestion
{
```

Reply:
Note that we're adding this to the JSON extension rather than replacing the whole file. (Maybe "Add this extension to the console-extensions.file.")